packages/eds-core-react/CHANGELOG.md

```diff
@@ -5,6 +5,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.18.0] - 2022-02-17
+
+### Added
+
+ - Added support for `18`px `Icon` size in preparation for compact mode ([#1827](https://github.com/equinor/design-system/issues/1827))
+ - `Button`: Enabled support for multiple lines and relative scaling by exposing a multitude of css-variables (see storybook for example of how this can be done)([#1858](https://github.com/equinor/design-system/issues/1858))
```

Comment:
I suggest we dont "release" the css-variables part just yet. Its still very experimental and subject to change because we haven't decided on naming convention or granularity.

Reply:
It was also only added specially just for envis

Reply:
As per the teams meeting we decided to try and move multiline button and table cell stories to eds-lab-react and not mention it in the release notes here